### PR TITLE
Update ci-cd-integration.md

### PR DESCRIPTION
**Pull request by michaelhigham**
Added documentation for the creation of HTTP/Browser monitors using the API.

### CODE REVIEW

File: docs/synthetics/ci-cd-integration.md [hunk 1]
```diff
@@ -122,6 +122,631 @@ AllLocations    276 us-west-1   passed  3/3 Passed  https://apps.sematext.com/ui
...
+### API Reference: conditions
+| Key | Type | Value | Description | Required
+| --- | --- | --- | --- | --- |
+| id | INTEGER | User-defined | Alert condition line number | YES |
+| type | STRING | -- | Alert condition type | YES |
+| | | METRIC | Metric | |
+| | | RESPONSE_BODY | Response Body | Only HTTP monitor |
+| | | RESPONSE_BODY_JSON | Response Body JSON | Only HTTP monitor |
+| | | RESPONSE_HEADER | Response Header | Only HTTP monitor |
+| | | RESPONSE_CODE | Response Code | Only HTTP monitor |
```
Comment (hsalsem) [inline docs/synthetics/ci-cd-integration.md:248]: Perhaps "Only **for** HTTP monitor" would be better

File: docs/synthetics/ci-cd-integration.md [hunk 2]
```diff
@@ -122,6 +122,631 @@ AllLocations    276 us-west-1   passed  3/3 Passed  https://apps.sematext.com/ui
...
+| Contains | Error |
+| Does Not Contain | Metric |
+| | Error |
+| > | Metric |
+| < | Metric |
+| = | Metric |
+| | Error |
+| != | Metric |
+| | Error |
+| Starts With | Error |
```
Comment (hsalsem) [inline docs/synthetics/ci-cd-integration.md:329]: For some reason these tables aren't rendering properly in the viewer I'm using, whereas all the other ones show up fine. Probably an issue on my end, but might be worth double checking.

**Comment by otisg** [comment]
@michaelhigham should this really be a part of this particular page?  That doesn't feel right...

**Comment by michaelhigham** [comment]
There was already the section there regarding [running a monitor](https://sematext.com/docs/synthetics/ci-cd-integration/#run-monitor-api) using the API, so it seemed like the appropriate place to put it. It is a lot of documentation though, so it might be better on a separate page.

**Comment by michaelhigham** [comment]
There was already the section there regarding [running a monitor](https://sematext.com/docs/synthetics/ci-cd-integration/#run-monitor-api) using the API, so it seemed like the appropriate place to put it. It is a lot of documentation though, so it might be better on a separate page.

**Comment by otisg** [comment]
Yeah, see my specific suggestion above.

**Comment by michaelhigham** [comment]
Yes I think that would be better. I will close the pull request and do as you suggest.

File: docs/synthetics/ci-cd-integration.md [hunk 3]
```diff
@@ -122,6 +122,631 @@ AllLocations    276 us-west-1   passed  3/3 Passed  https://apps.sematext.com/ui
...
+--data-raw '{
+    "name": "Example browser monitor without script",
+    "interval": "10m",
+    "enabled": true,
+    "locations": [
+        1,
+        2
+    ],
+    "url": "https://www.google.com/",
+    "script": "// Doesn'\''t matter since it doesn'\''t use a script",
```
Comment (hsalsem) [inline docs/synthetics/ci-cd-integration.md:508]: I'd just leave this field blank - `"script": ""`. I wrote it this way just to showcase that it wasn't needed for this variant of Browser monitor.

**Comment by michaelhigham** [comment]
Closing the request as per the suggestions from Otis.

**Review by hsalsem — COMMENTED** [review]
Great work, detailed info with good examples. I left a few notes on how to potentially improve things.

**Pull request closed without merging**